netweaver/monitoring.sls

```diff
@@ -0,0 +1,30 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+prometheus_sap_host_exporter_pkg:
+  pkg.installed:
+    - name: prometheus-sap_host_exporter
+
+sap_host_exporter_exporter_service:
+  service.running:
+    - name: prometheus-sap_host_exporter
+    - enable: True
+    - restart: True
+    - require:
+      - pkg: prometheus_sap_host_exporter_pkg
+      - file: sap_host_exporter_configuration
+
+# for each node the instance number change so we need to adapt configuration
+{% for node in netweaver.nodes if host == node.host %}
```

Comment:
Where are you getting this `host` from?
I guess you will need to add `{% set host = grains['host'] %}` at the top of the file, right?